src/pages/home/HomeCard/HomeCard.js

```diff
@@ -0,0 +1,28 @@
+import { Card } from '@dhis2/ui'
```

Comment:
This component and its styles are copied from https://github.com/dhis2/sms-configuration-app/tree/master/src/sms-overview/HomeCard